data/views.py

```diff
@@ -0,0 +1,175 @@
+from django.contrib import messages
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.shortcuts import render
+from django.views.generic.edit import FormView
+
+from data.forms import SuggestionCohorteForm
+from data.models import SuggestionAction, SuggestionStatut
+
+
+class IsStaffMixin(LoginRequiredMixin):
+    def dispatch(self, request, *args, **kwargs):
+        if not request.user.is_staff:
+            return self.handle_no_permission()
+        return super().dispatch(request, *args, **kwargs)
+
+
+ACTION_TO_VERB = {
+    SuggestionAction.SOURCE_AJOUT: "ajoutera",
+    SuggestionAction.SOURCE_SUPPRESSION: "supprimera",
+    SuggestionAction.SOURCE_MISESAJOUR: "mettra à jour",
+}
+
+
+class SuggestionManagment(IsStaffMixin, FormView):
+    form_class = SuggestionCohorteForm
+    template_name = "data/dags_validations.html"
+    success_url = "/data/suggestions"
+
+    def form_valid(self, form):
+        # MANAGE search and display suggestion_cohorte details
+        suggestion_cohorte = form.cleaned_data["suggestion_cohorte"]
+        if self.request.POST.get("search"):
```

Comment:
```suggestion
        if self.request.POST.get("search"):
```

Voir pour mieux nommer le paramètre d'URL ici

Reply:
p-e pas idéal de le faire ici, voir pour le faire dans la méthode `post` de notre `FormView` peut-être ? 

Cf https://ccbv.co.uk/projects/Django/5.0/django.views.generic.edit/FormView/#post

Reply:
je garde ça pour plus tard